include/UI/UIView.h

```diff
@@ -108,14 +110,32 @@ class UIView
   glm::vec3 get_clip_min();
   glm::vec3 get_clip_max();
 
-  void set_chracter(char c);
+  void set_clip_min(glm::vec3 clip_min);
+  void set_clip_max(glm::vec3 clip_max);
+
+  void add_character(char c);
 
   void remove_character();
 
   void compute_new_histogram_view();
 
   void addTransferFunction();
```

Comment:
should this be changed to snake case for consistency?

Reply:
done